spec/lib/miq_preloader_spec.rb

```diff
@@ -45,8 +45,9 @@
       FactoryBot.create_list(:vm, 2, :ext_management_system => ems)
       emses = ExtManagementSystem.all.load
       vms   = Vm.where(:ems_id => emses.select(:id))
-      expect { preload(emses, :vms, vms) }.to make_database_queries(:count => 1)
 
+      # there is now two queries - rails 7, why?
+      expect { preload(emses, :vms, vms) }.to make_database_queries(:count => 2)
       expect { expect(emses.first.vms.size).to eq(2) }.not_to make_database_queries
```

Comment:
I'm content with this answer.

The most important part of this test is the second line: `not_to make_database_queries`

Reply:
So, as I was looking at these tests, I was thinking some were not really the test and it's a test smell that we have several failures with rails 7 we needed to fix across the tests.  Perhaps, we should have 1 failing test that we need to either mark pending, commented out with a TODO, etc. and remove the other assertions with comments that fail if they're testing the same things.

Reply:
For now, I just fixed the comments and made them todos